pandas/tests/reshape/concat/test_datetimes.py

```diff
@@ -291,66 +307,87 @@ def test_concat_NaT_series_dataframe_all_NaT(self, tz1, tz2):
 
 
 class TestTimezoneConcat:
-    def test_concat_tz_series(self):
+    def test_concat_tz_series(self, unit, unit2):
         # gh-11755: tz and no tz
-        x = Series(date_range("20151124 08:00", "20151124 09:00", freq="1h", tz="UTC"))
-        y = Series(date_range("2012-01-01", "2012-01-02"))
+        x = Series(
+            date_range(
+                "20151124 08:00", "20151124 09:00", freq="1h", tz="UTC", unit=unit
+            )
+        )
+        y = Series(date_range("2012-01-01", "2012-01-02", unit=unit2))
         expected = Series([x[0], x[1], y[0], y[1]], dtype="object")
         result = concat([x, y], ignore_index=True)
         tm.assert_series_equal(result, expected)
 
-    def test_concat_tz_series2(self):
+    def test_concat_tz_series2(self, unit):
```

Comment:
i think can skip this one since it doesnt impact the result

Reply:
True.